datalake-dr-connector/src/main/java/com/sequenceiq/cloudbreak/datalakedr/converter/GrpcStatusResponseToDatalakeBackupRestoreStatusResponseConverter.java

```diff
@@ -11,31 +14,143 @@
 @Component
 public class GrpcStatusResponseToDatalakeBackupRestoreStatusResponseConverter {
 
+    static final String FAILED_STATE = "FAILED";
+
+    static final String STOP_SERVICES = "Stop services";
+
+    static final String START_SERVICES = "Start services";
+
+    static final String HBASE = "HBase";
+
+    static final String HBASE_ATLAS_JANUS = HBASE + " atlasJanus";
+
+    static final String HBASE_ATLAS_AUDIT = HBASE + " atlasEntityAudit";
+
+    static final String SOLR = "Solr";
+
+    static final String SOLR_DELETE = "Solr delete";
+
+    static final String SOLR_EDGE_INDEX = SOLR + " edge_index";
+
+    static final String SOLR_FULLTEXT_INDEX = SOLR + " fulltext_index";
+
+    static final String SOLR_RANGER_AUDITS = SOLR + " ranger_audits";
+
+    static final String SOLR_VERTEX_INDEX = SOLR + " vertex_index";
```

Comment:
I would rather introduce an enum for the Strings that are only used as a key for `getFailure()`, it might be a little cleaner, what do you think?

Reply:
I've created an `OperationEnum` class, let me know if that looks like what you meant.

Reply:
thanks, perfect!